modules/caddyhttp/reverseproxy/fastcgi/caddyfile.go

```diff
@@ -341,6 +341,7 @@ func parsePHPFastCGI(h httpcaddyfile.Helper) ([]httpcaddyfile.ConfigValue, error
 		rewriteMatcherSet := caddy.ModuleMap{
 			"file": h.JSON(fileserver.MatchFile{
 				TryFiles:  tryFiles,
+				TryPolicy: "first_exist_fallback",
```

Comment:
This could be risky if users configure their own `try_files` with a different last item. I think we should set `first_exist_fallback` only if the last item ends with `.php` maybe.